OrbitGl/TracepointCustom.h

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) 2020 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef ORBIT_TRACEPOINTCUSTOM_H
+#define ORBIT_TRACEPOINTCUSTOM_H
+
+#include "tracepoint.pb.h"
+
+namespace internal {
```

Comment:
By convention stuff from internal namespaces should only occur in in the header where it is defined (or the related .cc). If you need to make these functors more widely  available remove the "namespace internal {".

Reply:
Sure. Thank you so much! I will modify it right now.